Amazon.IonHashDotnet.Tests/IonHashWriterTest.cs

```diff
@@ -83,7 +83,7 @@ public void TestMiscMethods()
 
             //Commented out following assertion because it is failing
             //https://github.com/amzn/ion-hash-dotnet/issues/5
-            //Assert.AreEqual("null [5] null {hello:ion::hash::world}", stringWriter);
+            Assert.AreEqual("null [5] null {hello:ion::hash::'world'}", stringWriter.ToString());
```

Comment:
Interesting to note that the symbol `world` is being wrapped in single quotes.  It's not wrong, but it is something we should address in ion-dotnet.  I've opened https://github.com/amzn/ion-dotnet/issues/87 to track this